Vizzuality/heco-invest
# Let 165 sign up form

Add sign-up form page

## Testing instructions

* Respects the [visual design](https://www.figma.com/file/KC49uMR9t1YARmFR0Sr6aN/HeCo---Work-in-progress?node-id=742%3A16843)

* Contains fields for: first name, last name, email, password, confirm password, accepting the terms and conditions and privacy policy (all mandatory)

* The fields are validated:

    * required fields are not empty

    * email is valid

    * passwords match

* Errors are displayed if:

    * the validation does not pass

    * the user already exists

* The user’s locale is sent to the API (this is necessary to send the email verification in the correct language)

* Accessible by [WCAG 2.1 standards](https://www.w3.org/TR/WCAG21/)

* The fields are part of a form

* Input and form errors are correctly presented to assistive technologies (AT) ([see this guide on form validation](https://webaim.org/techniques/formvalidation/))

* Available for translation in Spanish, Portuguese and English

* Page works well on mobile devices (down to iPhone 5)

## Tracking

[LET-165](https://vizzuality.atlassian.net/browse/LET-165)
